pkg/k8s/ignore.go

```diff
@@ -37,41 +36,36 @@ func (i *IgnoreWrapper) AddIgnoredApps(app ...string) {
 	i.Apps = append(i.Apps, app...)
 }
 
-// LabelSelector returns the labels.Selector image of the ignored object.
-func (i *IgnoreWrapper) LabelSelector() (labels.Selector, error) {
-	sel := labels.Everything()
-
-	r, err := labels.NewRequirement("app", selection.NotIn, i.Apps)
+// IsIgnored returns true if the object events should be ignored.
+func (i *IgnoreWrapper) IsIgnored(obj interface{}) bool {
+	accessor, err := meta.Accessor(obj)
 	if err != nil {
-		return nil, err
+		return false
```

Comment:
In case of an error, we have to return `true` here, right?

```suggestion
		return true
```